src/styles/blocks/container.scss

```diff
@@ -0,0 +1,35 @@
+.container {
+  display: grid;
+
+  @for $i from 1 through 6 {
+    & :nth-child(#{$i})::after {
+      content: "#{$i}";
+    }
+  }
+
+  $colors-set:
+    1 "rgb(255, 0 , 0)",
+    2 "rgb(204, 0 , 0)",
+    3 "rgb(153, 0 , 0)",
+    4 "rgb(102, 0 , 0)",
+    5 "rgb(51, 0 , 0)",
+    6 "rgb(0, 0 , 0)";
```

Comment:
Should be in the variables file. 
```suggestion

```

Reply:
It is not strictly necessary in the case of small project like this, but preferable.